components/NavBar.vue

```diff
@@ -6,11 +6,11 @@
       app
       flat
       height="85"
-      class="mainNav bgViolet"
+      class="mainNav bgBurgundy"
     >
       <v-toolbar-title>
-        <NuxtLink class="link-box px-3" to="/">
-          <Logo alternative white has-custom-height custom-height="65" />
+        <NuxtLink class="link-box px-3 ml-6git" to="/">
+          <Logo alternative isNavBar has-custom-height custom-height="65" />
```

Comment:
O Vue tem umas convenções meio malucas: quando você cria uma propriedade, ela deve estar em [_Camel Case_](https://pt.wikipedia.org/wiki/CamelCase), mas, quando você "chama" essa propriedade no componente (como é o caso dessa linha), ela deve estar em [_Kebab Case_](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case).


```suggestion
          <Logo alternative is-nav-bar has-custom-height custom-height="65" />
```